dotCMS/src/main/java/com/dotcms/api/content/VanityUrlAPI.java

```diff
@@ -0,0 +1,68 @@
+package com.dotcms.api.content;
+
+import com.dotcms.content.model.VanityUrl;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.liferay.portal.model.User;
+
+import java.util.List;
+
+/**
+ * This API provides access to the information related to Vanity URLs 
+ * in dotCMS. Vanity URLs are alternate reference paths to
+ * internal or external URL's. Vanity URLs are most commonly used to give
+ * visitors to the website a more user-friendly or memorable way of reaching an
+ * HTML page or File, that might actually live “buried” in a much deeper path.
+ * 
+ * @author oswaldogallango
+ * @version 4.2.0
+ * @since June 12, 2017
+ */
+public interface VanityUrlAPI {
+	/**
+	 * This method load in cache all the active Vanity URLs 
+	 * created in dotCMS
+	 */
+	void initializeActiveVanityURLsCache(final User user);
```

Comment:
We need to use spaces instead tabs, seems like this is new code and it has a mix of spaces and tabs (not only in this file)